examples/Demo/Shared/Pages/Grid/GridPage.razor

```diff
@@ -54,6 +54,12 @@
     To display an element only on a given range of screen sizes, you can combine a these values.
     e.g. <code>GridItemHidden.Md | GridItemHidden.Lg</code> will display the element for all screen sizes, except on medium and large devices.
 </p>
+<p>
+    <b>AdaptiveRendering</b> allows you not to generate the HTML code of the <b>FluentGridItem</b> that respects the <code>HiddenWhen</code>.
+    In other words, when <code>AdaptiveRendering=false</code> (default), the content of the <b>FluentGridItem</b> is simply hidden by CSS styles,
+    whereas if <code>AdaptiveRendering=true</code>, the content of the <b>FluentGridItem</b> is not rendered by Blazor.
+    In certain cases where this content is important, it may be more efficient not to generate the HTML code instead of simply hiding it.
```

Comment:
This allows for fine-grained control over when HTML is generated or not, for example in a case where rendering the grid item takes a lot of time or leads to a lot of data being transferred.